### ISSUE TITLE
Introspection or GraphiQL don't like interface types

### ISSUE THREAD

**Issue by craig-day** [absinthe-graphql/absinthe_plug#123]
After upgrading to v1.4, it seems that GraphiQL does not like interface types anymore. The type is missing from the schema docs and the IDE claims the type doesn't exist, but the query will successfully execute. 

![](https://content.screencast.com/users/CDay.zendesk/folders/Jing/media/382f4ddb-c60a-44f4-a437-17ad28b9de5b/00000518.png)

ticket schema: 

```
object :ticket do
  field :id, :integer

  field :test_iface, :iface_test do
    resolve fn _, _ ->
      {:ok, %{type: "one", name: "foo", email: "bar", custom: "custom"}}
    end
  end
end
```

iface_test schema: 

```
object :one do
  field :name, :string
  field :email, :string

  field :custom, :string

  interface :iface_test
end

object :two do
  field :name, :string
  field :email, :string

  field :unique, :string

  interface :iface_test
end

interface :iface_test do
  field :name, :string
  field :email, :string

  resolve_type fn
    %{type: "one"}, _ -> :one
    %{type: "two"}, _ -> :two
    _, _ -> :one
  end
end
```

I don't think I'm missing anything, and this same schema worked on v1.3. Let me know if you need anything else!

**Comment by bruce** [comment]
Which Absinthe version are you using? Have you tried v1.4.1?

**Comment by tlvenn** [comment]
@bruce , should we create an issue on absinthe project to track it there ?

**Comment by benwilson512** [comment]
Fixed in Absinthe v1.4.3.
